docs/AboutUs.md

```diff
@@ -24,26 +24,26 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 <img src="images/johndoe.png" width="200px">
 
 [[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[portfolio](team/yuqitanyq.md)]
 
 * Role: Team Lead
 * Responsibilities: UI
 
-### Johnny Doe
+### Tan Yu Qi
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/yuqitanyq.png" width="200px">
 
-[[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
+[[github](http://github.com/yuqitanyq)] [[portfolio](team/yuqitanyq.md)]
```

Comment:
Take care to make sure that there won't be refactoring issues when we change links.